UnityProject/WhileYoureHere/Assets/Scripts/radio interaction/RadioData.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "RadioData", menuName = "Scriptable Objects/RadioData")]
```

Comment:
Remove the spaces here, this will create another entry in the dropdown menu when you create a new scriptableobject

```suggestion
[CreateAssetMenu(fileName = "RadioData", menuName = "ScriptableObjects/RadioData")]
```